python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -128,10 +133,7 @@ def run_transformer_int4(repo_id,
                                                      trust_remote_code=True, optimize_model=False)
         # Need to use LlamaTokenizer, reason please refer to issue: https://github.com/intel-analytics/BigDL/issues/8944
         tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
-    elif repo_id in ['meta-llama/Llama-2-7b-chat-hf','meta-llama/Llama-2-13b-chat-hf',
-                     'meta-llama/Llama-2-70b-chat-hf','decapoda-research/llama-7b-hf',
-                     'decapoda-research/llama-65b-hf','lmsys/vicuna-7b-v1.5',
-                     'lmsys/vicuna-13b-v1.3','project-baize/merged-baize-30b']:
+    elif repo_id in LLAMA_IDS:
```

Comment:
maybe we can also remove L130 ? Other LGTM

Reply:
Have removed `elif repo_id in ['meta-llama/Llama-2-70b-chat-hf']:`